workspace/__lib__/xod/uart/uart-usb/patch.cpp

```diff
@@ -0,0 +1,12 @@
+struct State {
+    uint8_t mem[sizeof(HardwareUart)];
+    HardwareUart* uart;
+};
+
+{{ GENERATED_CODE }}
+
+void evaluate(Context ctx) {
+    auto state = getState(ctx);
+    state->uart = new (state->mem) HardwareUart(SerialUSB, (uint32_t) getValue<input_BAUD>(ctx));
```

Comment:
I guess it will fail on Leonardo because `SerialUSB` is `Serial_` and not `HardwareSerial` there.